packages/suite/src/views/wallet/send/components/Outputs/index.tsx

```diff
@@ -35,27 +40,62 @@ const Row = styled.div`
     }
 `;
 
-const Outputs = () => {
+interface Props {
+    disableAnim?: boolean; // used in tests, with animations enabled react-testing-library can't find output fields
+}
+
+const Outputs = ({ disableAnim }: Props) => {
     const { outputs } = useSendFormContext();
+    const [renderedOutputs, setRenderedOutputs] = useState(1);
+    const lastOutputRef = useRef<HTMLDivElement | null>(null);
+
+    const onAddAnimationComplete = () => {
+        // scrolls only on adding outputs, doesn't scroll on removing them
+        if (outputs.length > 1 && outputs.length > renderedOutputs) {
+            lastOutputRef?.current?.scrollIntoView({ behavior: 'smooth' });
+        }
+
+        setRenderedOutputs(outputs.length);
+    };
+
+    useEffect(() => {
+        if (outputs.length < renderedOutputs) {
+            // updates rendered outputs count when removing an output
+            // this is necessary because onAddAnimationComplete is not fired when removing 2nd output
+            setRenderedOutputs(outputs.length);
+        }
+    }, [outputs.length, renderedOutputs, setRenderedOutputs]);
+
+    const WrapperComponent = disableAnim ? Wrapper : AnimatePresence;
     return (
-        <Wrapper>
+        <WrapperComponent initial={false}>
             {outputs.map((output, index) => (
-                <OutputWrapper key={output.id} index={index}>
-                    {output.type === 'opreturn' ? (
-                        <OpReturn outputId={index} />
-                    ) : (
-                        <>
-                            <Row>
-                                <Address outputId={index} outputsCount={outputs.length} />
-                            </Row>
-                            <Row>
-                                <Amount outputId={index} />
-                            </Row>
-                        </>
-                    )}
-                </OutputWrapper>
+                <motion.div
+                    {...(outputs.length > 1 && !disableAnim ? ANIMATION.EXPAND : {})} // do not animate if there is only 1 output, prevents animation on clear
```

Comment:
for cleaner code
create `const animation = outputs.length && !disableAnim ? ANIMATION.EXPAND : {};`
and use here:
`<motion.div {...animation}>`